internal/classification-rubric.md

```diff
@@ -0,0 +1,20 @@
+# Classification Rubric
+
+This is an initial set of guidelines to use
+when deciding on the classification of
+[fields for API/SDK resources](./adding-a-new-field-readme.md#add-new-fields-to-the-apisdk-resource-protobufs).
```

Comment:
I am thinking it might be a good idea to explicitly call out that this classification is for auditing purposes?